profile_bluesky/startup/instrument/devices/cyberstar.py

```diff
@@ -0,0 +1,47 @@
+"""
+APD detector
+"""
+
+__all__ = ['cyberstar_mag_parameters', 'cyberstar_8c_parameters']
+
+from ..session_logs import logger
+logger.info(__file__)
+
+from ophyd import Component, Device, EpicsSignal, EpicsSignalRO
+from ..framework import sd
+
+
+class CyberstarDevice(Device):
+
+    hv_readback = Component(EpicsSignalRO, 'HV.VAL', kind='config',
```

Comment:
[`ophyd.areadetector.base.EpicsSignalWithRBV`](https://github.com/bluesky/ophyd/blob/master/ophyd/areadetector/base.py) might be good to use here.  I believe it is possible to use this PV naming scheme with that Device.